apps/web-giddh/src/app/ledger/ledger.component.scss

```diff
@@ -1,5 +1,68 @@
 @import "../../assets/css/color.scss";
+.wrap-transaction-map{
+    border:1px solid $gainsboro;
+    border-bottom: none !important;
+    .ledger-book{
+        box-shadow: none;
+        .left-col-new{
+            border-right: 1px solid $gainsboro;
+        }
+        .checkbox-span{
+            height: 49px;
+            min-height: 49px;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+        .ledger-row{
+            &:hover{
+                background-color: transparent;
+            }
+        }
+    }
+}
+#ledger-transaction-left-list , #ledger-transaction-right-list {
+    td{
+        padding: 0 !important;
+        &:first-child{
+            border-bottom: none !important;
+        }
+        .d-flex.align-item-center{
+            align-items: stretch !important;
+        }
+        input, sh-select, .form-control{
+            min-height: 49px;
+            height: 49px;
+            border:0 !important;
+
+        }
+        input[type="checkbox"]{
+            min-height: 20px;
+            height: 20px;
+        }
+
+    }
+    sh-select::ng-deep input{
```

Comment:
`::ng-deep` is deprecated, please refer the doc for reference: https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

> The shadow-piercing descendant combinator is deprecated and support is being removed from major browsers and tools. As such we plan to drop support in Angular (for all 3 of /deep/, >>> and ::ng-deep). Until then ::ng-deep should be preferred for a broader compatibility with the tools.